src/main/java/bio/terra/workspace/service/resource/reference/ReferenceGcsBucketAttributes.java

```diff
@@ -0,0 +1,17 @@
+package bio.terra.workspace.service.resource.reference;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class ReferenceGcsBucketAttributes {
```

Comment:
can we combine this with the controlled counterpart?

Reply:
It is a good question. If it turns out that the attributes are always alike for referenced and controlled, then we should make an attributes package and just share them. OTOH, if we change one and not the other, it would be nice to have different classes serialized in Stairway. Hmm...